HelpSource/Classes/UnixFILE.schelp

```diff
@@ -59,6 +59,9 @@ read four bytes and return as a link::Classes/Float::.
 method::getDouble
 read eight bytes and return as a link::Classes/Float::.
 
+method::getPascalString
+Reads the next byte as an unsigned integer N, then reads the following N bytes and returns them as a link::Classes/String:.
```

Comment:
broken link. (sorry, i would fix it myself but i'm having trouble pushing to your branch)